src/lua/widget/container.c

```diff
@@ -88,7 +88,7 @@ static int child_added(lua_State *L)
   lua_widget widget;
   luaA_to(L, lua_widget,&widget,2);
   lua_getuservalue(L,1);
-  luaA_push(L,lua_widget,&widget);
+  lua_pushlightuserdata(L, widget->widget);
```

Comment:
arn't these two pointers aliased via the type system , lua/widget.c line 102 aliases the gtk widget to the lua widget...